homeworks/12-mars-viewer-mobx/src/store/favouritesStore.tsx

```diff
@@ -0,0 +1,28 @@
+import { action, makeAutoObservable } from 'mobx';
+
+class FavouritesStore {
+  favourites: string[] = []
+
+  constructor() {
+    makeAutoObservable(this, { addFavourite: action.bound, deleteFavourite: action.bound });
+  }
+
+  addFavourite(id: string) {
+    this.favourites.push(id);
+  }
+
+  deleteFavourite(id: string) {
+    // console.log(this.favourites, id);
```

Comment:
This should be deleted, production code shouldn't contain comments of unused code